lib/commons/aria/get-element-unallowed-roles.js

```diff
@@ -22,6 +17,11 @@ const dpubRoles = [
   'doc-noteref'
 ];
 
+const landmarkRoles = {
```

Comment:
nit: aren't there other landmark roles? What is special about these two?

Reply:
These are the only two elements that change role depending on if they are in a landmark. If you do `main > header`, the header element is not a `banner`. If you just do `body > header` it is.

This causes the odd behavior that `body > header[role=banner]` passes, but `body > main > header[role=banner]` fails. That last case should fail for being a nested landmark (which it does), but not for having a role on an element that doesn't allow it.

Reply:
yeah, I was commenting on the name of the variable, makes it seem like it is all the landmark roles